pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -5,7 +5,10 @@
 import pyutilib.th as unittest
 from pyomo.contrib.mindtpy.tests.eight_process_problem import \
     EightProcessFlowsheet
-from pyomo.contrib.mindtpy.tests.MINLP_simple import SimpleMINLP
+from pyomo.contrib.mindtpy.tests.MINLP_simple import SimpleMINLP as SimpleMINLP
+from pyomo.contrib.mindtpy.tests.MINLP2_simple import SimpleMINLP as SimpleMINLP2
+from pyomo.contrib.mindtpy.tests.MINLP3_simple import SimpleMINLP as SimpleMINLP3
```

Comment:
is there no better name for these?

Reply:
It's what they were already named - I didn't implement them myself. I agree that there could be better names

Reply:
Someone can rename them but I'm not up for it right now